types/mux-embed.d.ts

```diff
@@ -548,6 +548,16 @@ declare module 'mux-embed' {
 
   export const events: MuxEvents;
 
+  type MuxType = (task: any) => void;
+  type muxProps = {
+    utils: publicUtils;
+    monitor: typeof monitor;
+  };
+  interface MuxGlobal extends MuxType, muxProps {}
+
+  export var mux: MuxGlobal;
+  export default mux;
```

Comment:
some minimal typings to get a default export based on the types in mux-embed itself.